ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -571,6 +571,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-fullyprivate-internal-registry-amd-f14
```

Comment:
missing job to cover tests for destructive and arm platform, suggest to add below two jobs:
- azure-ipi-fullyprivate-internal-registry-amd-f28-destrcutive
- azure-ipi-fullyprivate-internal-registry-arm-f14